adobe/spectrum-web-components
# chore: use latest Spectrum CSS versions

## Description
- update Windows High Contrast Mode support across the library by taking on the most recent Spectrum CSS versions

## Related issue(s)

- fixes #2229

## How has this been tested?

    1. Go [here](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/action-button-static-white-quiet--s)
    2. Also go [here](https://spectrum-css--spectrum-web-components.netlify.app/storybook/?path=/story/action-button-static-white-quiet--s)
    3. See which one did it better.
    1. Using Assistive Labs for Window High Contrast Mode
    2. Go [here](https://spectrum-css--spectrum-web-components.netlify.app/storybook/?path=/story/action-button-static-black-quiet--s) and all the other Action Button Stories
    3. See that they appear like: <img width="192" alt="image" src="https://user-images.githubusercontent.com/1156657/170152384-8dd5d104-7531-423a-bdfe-ea4e44ded403.png">
    1. Using Assistive Labs for Window High Contrast Mode
    2. Go [here](https://spectrum-css--spectrum-web-components.netlify.app/storybook/?path=/story/button-accent-fill--default) and all the other Button Stories w/ disabled buttons
    3. See that they appear like: <img width="298" alt="image" src="https://user-images.githubusercontent.com/1156657/170152513-8cb0c83e-7b06-4136-825c-c3e15326b172.png">

## Types of changes

## Checklist
